third_party/intel/lib/TritonIntelGPUToLLVM/TritonOpsToLLVM.cpp

```diff
@@ -121,11 +121,22 @@ class LoadStorePrefetchOpConversion
            "only support 1d/2d load/store/prefetch for now");
 
     unsigned dataSize = tensorType.getElementType().getIntOrFloatBitWidth();
-    unsigned blockWidth = tensorType.getShape()[1];
-    assert(blockWidth == 16 || blockWidth == 32 && "only support 16/32 block");
-    unsigned vBlks = blockWidth == 32 ? 2 : 1;
-    blockWidth = 16;
     unsigned blockHeight = tensorType.getShape()[0];
+    unsigned blockWidth = tensorType.getShape()[1];
+    assert(blockWidth == 16 || blockWidth == 32 ||
+           blockWidth == 64 && "only support 16/32/64 block");
```

Comment:
```suggestion
    assert((blockWidth == 16 || blockWidth == 32 ||
           blockWidth == 64) && "only support 16/32/64 block");
```
I think the current assert is incorrect